common/fs/src/cache/mod.rs

```diff
@@ -203,10 +215,26 @@ impl FileSystem {
             acc
         };
 
-        let events = events_stream
-            .into_stream()
+        let resume_events_recv = {
+            let _fs = fs.try_lock().expect("couldn't lock filesystem cache");
+            _fs.resume_events_recv
+                .clone()
+                .map({
+                    let fs = fs.clone();
+                    move |(inode, event_time)| {
+                        fs.try_lock()
+                            .expect("couldn't lock filesystem cache")
+                            .wd_by_inode
+                            .get(&inode)
+                            .map(|wd| (Ok(WatchEvent::Modify { wd: wd.clone() }), event_time))
```

Comment:
This is a cool idea.

What would happen for deletes with lots of writes?

I imagine that the event order would be the following:

- Modify
  - Produces a Resume Modify event to the tail of the queue
- Modify (ignored)
- Modify (ignored)
- Delete
- Resume Modify event